cmd/dependabot/dependabot_test.go

```diff
@@ -12,7 +12,7 @@ import (
 )
 
 func TestDependabot(t *testing.T) {
-	err := exec.Command("go", "build", "dependabot.go").Run()
+	err := exec.Command("go", "build", ".").Run()
```

Comment:
When you build with a file, the version info isn't injected.

Reply:
Ah, I remember hitting something like this at a prior employer. It was a royal pain to workaround.